examples/servlet/README.md

```diff
@@ -16,3 +16,12 @@ A simple example application to show how Bugsnag works in a Servlet-based Java a
 - Cause a crash
 
     http://localhost:8080/servlet
+
+## Running in Docker
+
+Execute the following commands in the root of the repository:
+
+```
+docker-compose build
+docker-compose up servlet
```

Comment:
the container ended immediately for me after building this so I was unable to load the servlet. Do we need to do something to stop gradle exiting?